packages/flutter/lib/src/material/pickers/date_range_picker_dialog.dart

```diff
@@ -0,0 +1,678 @@
+        );
+        break;
+
+      case DatePickerEntryMode.input:
+        contents = _InputDateRangePickerDialog(
+          selectedStartDate: _selectedStart,
+          selectedEndDate: _selectedEnd,
+          picker: Container(
+            padding: const EdgeInsets.symmetric(horizontal: 24),
+            height: orientation == Orientation.portrait
+              ? _inputFormPortraitHeight
+              : _inputFormLandscapeHeight,
+            child: Column(
+              children: <Widget>[
+                const Spacer(),
+                InputDateRangePicker(
+                  key: _inputPickerKey,
+                  initialStartDate: _selectedStart,
+                  initialEndDate: _selectedEnd,
+                  firstDate: widget.firstDate,
+                  lastDate: widget.lastDate,
+                  onStartDateChanged: _handleStartDateChanged,
+                  onEndDateChanged: _handleEndDateChanged,
+                  autofocus: true,
+                  autovalidate: _autoValidate,
+                  helpText: widget.helpText,
+                  errorInvalidRangeText: widget.errorInvalidRangeText,
+                  errorFormatText: widget.errorFormatText,
+                  errorInvalidText: widget.errorInvalidText,
+                  fieldStartHintText: widget.fieldStartHintText,
+                  fieldEndHintText: widget.fieldEndHintText,
+                  fieldStartLabelText: widget.fieldStartLabelText,
+                  fieldEndLabelText: widget.fieldEndLabelText,
+                ),
+                const Spacer(),
+              ],
+            ),
+          ),
+          onConfirm: _handleOk,
+          onCancel: _handleCancel,
+          onToggleEntryMode: _handleEntryModeToggle,
+          confirmText: widget.confirmText ?? localizations.okButtonLabel,
+          cancelText: widget.cancelText ?? localizations.cancelButtonLabel,
+          // TODO(darrenaustin): localize 'SELECTED DATE RANGE'
+          helpText: widget.helpText ?? 'SELECTED DATE RANGE',
+        );
+        size = orientation == Orientation.portrait ? _inputPortraitDialogSize : _inputLandscapeDialogSize;
+        insetPadding = const EdgeInsets.symmetric(horizontal: 16.0, vertical: 24.0);
+        shape = const RoundedRectangleBorder(
+          borderRadius: BorderRadius.all(Radius.circular(4.0))
+        );
+        break;
+    }
+
+    return Dialog(
+      child: AnimatedContainer(
+        width: size.width,
+        height: size.height,
+        duration: _dialogSizeAnimationDuration,
+        curve: Curves.easeIn,
+        child: MediaQuery(
+          data: MediaQuery.of(context).copyWith(
+            textScaleFactor: textScaleFactor,
+          ),
+          child: Builder(builder: (BuildContext context) {
+            return contents;
+          }),
+        ),
+      ),
+      insetPadding: insetPadding,
+      shape: shape,
+      clipBehavior: Clip.antiAlias,
+      elevation: 24.0,
```

Comment:
Dialog defaults to 24, I wonder if this should be left out so that when a user modifies the `DialogTheme`'s elevation, it applies to the pickers too (same comment probably applies to the already merged DatePicker)

Reply:
Yeah, I just caught this when I was looking at the shape. I will fix this shortly and double check the Date Picker (but I am pretty sure I already deferred this to theme for that one). Thx.